### DISCUSSION TITLE
Server Setup for 1.20.1

### DISCUSSION THREAD

**Discussion by JBar2K** [Luke100000/minecraft-comes-alive#982] [Q&A]
When starting server, this error stops the process; "Failed to start the minecraft server
net.minecraftforge.fml.LoadingFailedException: Loading errors encountered: [
        Mod ?emca?r requires ?6architectury?r ?o9.0.8 or above?r
?7Currently, ?6architectury?r?7 is ?o?nnot installed"

Using forge version 1.20.1-47.3.0
MCA 7.5.19+1.20.1-universal
Architectury 13.0.6-neoforge

I have placed the architectury file in the mods folder, the main folder where the run.bat file is, and a few other places. 

Any recommendations?

**Comment by Luke100000** [comment]
Architectury 13.0.6 is for 1.21

**Comment by JBar2K** [comment]
Thank you. All is fine now! Your mod is impressive.
